lisa/nic.py

```diff
@@ -151,14 +152,42 @@ def get_unpaired_devices(self) -> List[str]:
         return [x.upper for x in self.nics.values() if not x.lower]
 
     def get_upper_nics(self) -> List[str]:
-        return [x.upper for x in self.nics.values()]
+        return list(self.nics.keys())
+
+    def get_paired_devices(self) -> List[str]:
```

Comment:
how about `get_upper_with_secondary_nic`? The secondary mean the "master" nic has a "slave" nic.